challenges/classes.js

```diff
@@ -1,7 +1,47 @@
 // 1. Copy and paste your prototype in here and refactor into class syntax.
+class CuboidMaker {
+    constructor(attrs) {
+        this.length = attrs.length
+        this.width = attrs.width
+        this.height = attrs.height
+    }
+    volume() {
+        return 'Volume: ' + this.length * this.width * this.height
+    }
+    surfaceArea() {
+        return 'Surface Area: ' + 2 * (this.length * this.width + this.length * this.height + this.width * this.height)
+    }
+}
+
+class CubeMaker extends CuboidMaker {
+    constructor(CubeMakerAttrs) {
+        super(CubeMakerAttrs);
+    }
+    volume() {
+        return 'Volume: ' + (this.length * this.length * this.length);
+    }
+    surfaceArea() {
+        return 'Surface Area: ' + 6 * (this.length * this.length);
+    }
+}
+
+const cuboid = new CuboidMaker({
+    length: 4,
+    width: 5,
+    height: 5,
+});
+
+const cube = new CubeMaker({
+    length: 4,
+    width: 5,
+    height: 5,
```

Comment:
Not very important but to create a cube object you only need the length property. Just try to keep your code as minimal as possible